src/react-native/15-using-asyncstorage/using-asyncstorage.md

```diff
@@ -39,10 +67,59 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 <details>
 <summary>Click to see the solution</summary>
 
+✏️ Update **src/services/pmo/api/api.ts** to be:
+
+@diff ../../../exercises/react-native/15-async-storage/01-problem/src/services/pmo/api/api.ts ../../../exercises/react-native/15-async-storage/01-solution/src/services/pmo/api/api.ts only
+
+✏️ Update **/src/services/storage/storage.ts** to be:
+
+@diff ../../../exercises/react-native/15-async-storage/01-problem/src/services/storage/storage.ts ../../../exercises/react-native/15-async-storage/01-solution/src/services/storage/storage.ts only
+
+</details>
+
+## Objective 2: Migrating data between versions
+
+### Concept TODO
+
+TODO
+
+### Setup 2
+
+✏️ Create **/src/services/DataMigration/DataMigration.tsx** and update it to be:
+
+@sourceref ../../../exercises/react-native/15-async-storage/02-problem/src/services/DataMigration/DataMigration.tsx
+
+✏️ Create **/src/services/DataMigration/index.ts** and update it to be:
+
+@sourceref ../../../exercises/react-native/15-async-storage/02-problem/src/services/DataMigration/index.ts
+
+### Verify 2
+
+✏️ Create **/src/services/DataMigration/DataMigration.test.tsx** and update it to be:
+
+@sourceref ../../../exercises/react-native/15-async-storage/02-problem/src/services/DataMigration/DataMigration.test.tsx
+
+### Exercise 2
+
 TODO
```

Comment:
Please include notes for what they’re doing in the Exercise section. We included the requirements for the “breakdown” stories here: https://bitovi.atlassian.net/browse/LD-230